base_report_to_printer/tests/test_report.py

```diff
@@ -2,14 +2,13 @@
 # Copyright 2017 Tecnativa - Jairo Llopis
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-import mock
+from unittest import mock
 
 from odoo import exceptions
 from odoo.tests import common
 
 
-@common.at_install(False)
-@common.post_install(True)
+@common.tagged("post_install", "-at_install")
```

Comment:
Unless strictly required, it is a good rule of thumb to avoid post-install tests, as they fail on integration environments more often than others.

Is it possible to remove this decorator?